utility/src/main/java/org/oppia/util/parser/CustomBulletSpan.kt

```diff
@@ -0,0 +1,73 @@
+package org.oppia.util.parser
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.graphics.Path
+import android.graphics.Path.Direction
+import android.text.Layout
+import android.text.Spanned
+import android.text.style.LeadingMarginSpan
+import org.oppia.util.R
+
+/**
+ * Copy of [android.text.style.BulletSpan] from android SDK 28 with removed internal code.
+ * This class helps us to customise bullet radius, gap width and offset present in rich-text.
+ * Reference: https://github.com/davidbilik/bullet-span-sample
+ */
+class CustomBulletSpan(context: Context) : LeadingMarginSpan {
```

Comment:
It would be really nice to add tests for the drawing logic here, but I'm not quite sure.

Reply:
@BenHenning I am actually now sure how to do that or what test cases should be? Should we check whether the radius, gap, width is correct or not?

Reply:
I’m not actually sure, which is more why I said it would be nice. :) I think the correct thing to do is test the custom draw logic in different scenarios with a screen shot tester. Can you file an issue with the different test scenarios and add a TODO?

Reply:
Issue https://github.com/oppia/oppia-android/issues/562 added to TODO also in code.